dotcom-rendering/src/components/FootballMatchesPage.tsx

```diff
@@ -76,6 +76,10 @@ export const FootballMatchesPage = ({
 					right: 0;
 				}
 			}
+
+			${until.leftCol} {
+				padding-bottom: ${space[9]}px;
+			}
```

Comment:
I think this should be applied to all breakpoints (not just until leftCal) otherwise it'd look like this 

![image](https://github.com/user-attachments/assets/0bf7b1e1-c095-4717-a5a6-8f532a07b7f2)

Reply:
Okay yes, we can accept the double whitespace when the more button isn't there